main.cpp

```diff
@@ -21,8 +21,14 @@
 #include "mbed.h"
 #include "drivers/QSPI.h"
 
-#define CMD_WRITE          0x02
+// for Nordic platforms fast read should be used and the QSPI flash memory start address
+// along with the total buffer size need to be 4-byte aligned/divisible by 4
+#if TARGET_NORDIC
+#define CMD_READ           0x0B // Fast read
```

Comment:
I have a question regarding the TARGET_NORDIC. is all Nordic target should use the same fast read command?
I ask this because this will apply the change to all current and future Nordic boards and not only for nRF52x boards.
if you want to make this change only for the nRF52x boards you should use TARGET_NRF5x macro instead.